src/css/style.css

```diff
@@ -1,4 +1,91 @@
-/* Your custom styles go here! */
+:root {
+    --grey-lightest: #f7f9fa;
+    --blue-light: rgba(43, 63, 86, 0.81);
+    --green: #72cc18;
+    --green-light: rgba(114, 204, 24, 0.68);
+}
+
+html, body {
+    margin: 0;
+    height: 100%;
+    font-family: "Helvetica Neue", Sans-Serif, sans-serif;
+    background: var(--blue-light);
+    color: var(--grey-lightest);
+}
+
+body a {
+    color: orange;
+}
+
+body a:hover {
+    color: darkorange;
+}
+
+ul {
+    list-style-type: square;
+    list-style-position: inside;
+}
+
+#participants li {
+    border-bottom: solid 1px darkorange;
+    margin-bottom: 2px;
+    padding: 1px;
```

Comment:
Small nit: usually you want to use larger/consistent values for the layout that are multiples of a base number (e.g. it's common to use 4px as the smallest value, and then multiples of 4). Typically, more space to "breathe" (visually) is nicer to look at, so seeing 1px here makes me think you either don't need this because it's too small to make a difference, or it needs to be a bigger value. (e.g. `padding-bottom:4px` or something instead.)

Here's an article related to this idea: https://designary.com/tip/layout-basics-grid-systems-and-the-4px-grid/

Reply:
Thanks! Going with bottom padding of 4 as you suggest